include/net/fota_download.h

```diff
@@ -172,14 +171,14 @@ int fota_download(const char *host, const char *file, const int *sec_tag_list,
  *              download, both paths will be treated as upgradable bootloader slot 0
  *              and slot 1 binaries respectively, and only the binary corresponding to
  *              the currently inactive bootloader slot will be selected and downloaded.
-*               See <a href="https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/config_and_build/bootloaders/bootloader.html">
+ *              See <a href="https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/config_and_build/bootloaders/bootloader.html">
```

Comment:
I'm surprised that compliance is not complaining about this long line 🤔

Reply:
I've seen cases when long line is fine when there are no spaces etc. where you can split the line